users/brett/brett.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include QMK_KEYBOARD_H
+
+enum userspace_custom_keycodes {
+  PLACEHOLDER = SAFE_RANGE, // Can always be here
+  FAT_ARROW, // =>
+  SKINNY_ARROW, // ->
+  REVERSE_ARROW, // <-
+  CONCAT, // <>
+  MAP, // <$>
+  MAP_FLIPPED, // <#>
+  FLAP, // <@>
+  PIPE, // |>
+  ALT, // <|>
+  APPLY, // <*>
+  AND, // &&
+  OR, // ||
+  BIND, // >>=
+  BIND_FLIPPED, // =<<
+  VOID_LEFT, // <$
+  VOID_RIGHT, // $>
+  DOUBLE_COLON, // ::
+  FLASH, // Handle keyboard flashing
+  DYNAMIC_MACRO_RANGE // Should always be last
```

Comment:
This is no longer needed, as these keycodes are now part of the core keycodes. 

https://github.com/qmk/qmk_firmware/blob/5ab0eeb513f50a14e51467bf76a19997b75d65fb/quantum/quantum_keycodes.h#L508-L513

```suggestion
```